src/lib_c/x86_64-windows-msvc/c/sysinfoapi.cr

```diff
@@ -0,0 +1,54 @@
+require "c/winnt"
+require "c/win_def"
+require "c/int_safe"
+
+lib LibC
+  fun GetNativeSystemInfo = GetNativeSystemInfo(system_info : SYSTEM_INFO*)
+
+  PROCESSOR_ARCHITECTURE_AMD64   =      9
```

Comment:
These constants are not used anywhere.

Reply:
They're unrelated to this PR.

Reply:
Yeah, I can remove them.

I originally put them just in case we wanted to exposed a processor architecture function or something later which comes from the same `GetNativeSystemInfo` call, but I since such a function doesn't exist yet, the associated constants are just sort of hanging out as dead code.

Reply:
Removed constants.